### DISCUSSION TITLE
Deno APIs missing in Deno runtime

### DISCUSSION THREAD

**Discussion by kevgeoleo** [denoland/deno#24640] [Q&A] [answered]
Hi,

I am using deno 1.45.2.
I am trying to using Deno APIs like `Deno.listenDatagram` API (https://docs.deno.com/api/deno/~/Deno.listenDatagram) 
but I get "error: Uncaught (in promise) TypeError: Deno.listenDatagram is not a function
const listener1 = Deno.listenDatagram({` "

Is it a problem with the deno version or is the API deprecated?

Thanks

**Comment by bartlomieju** [comment, marked as answer]
This is an unstable API, you need to pass `--unstable` or `--unstable-net` flag to enable this API.
